_topic_map.yml

```diff
@@ -221,16 +229,17 @@ Topics:
 ---
 Name: Support
 Dir: support
-Distros: openshift-enterprise,openshift-webscale,openshift-online,openshift-dedicated
+Distros: openshift-enterprise,openshift-webscale,openshift-online,openshift-dedicated,openshift-origin
```

Comment:
@vrutkovs is this support section valid for origin?

Reply:
It is, especially "Gathering info"